SmartVision/FaceAnalysis/JSON/AnalyzedFaces.cs

```diff
@@ -134,12 +135,17 @@ public struct Attributes
         public Age age { get; set; }
     }
 
-    public struct FaceRectangle
+    public class FaceRectangle
     {
-        public int width { get; set; }
-        public int top { get; set; }
-        public int left { get; set; }
-        public int height { get; set; }
+        public int Width { get; set; }
```

Comment:
Is it ok that you renamed these properties? Because in JSON response they all start with lower case